requirements.txt

```diff
@@ -1,4 +1,4 @@
-boto3~=1.18.40
+boto3~=1.26.90
 python-json-logger~=0.1
 dicttoxml~=1.7.4
 cumulus-message-adapter-python~=2.1.0
```

Comment:
Need to work with new version of cumulus-message-adapter-python

Reply:
Do we have ticket for cumulus core to verify that all the updated packages work?

Reply:
cma-python is also awaiting merge (approved by jennifer but some things requested that I'm going to follow up on)
cumulus core is awaiting these, that's the main ticket 3270